provider/cmd/pulumi-resource-command/schema.json

```
@@ -191,8 +191,12 @@
                     "$ref": "#/types/command:remote:Connection"
                 },
                 "localPath": {
-                    "description": "The path of the file to be copied.",
-                    "type": "string"
+                    "description": "The file/folder to be copied.\nIf the input is an Asset, it will be interpreted as the contents of a file.\nIf the input is an Archive, it will be interpreted as the contents of a folder.\nIf the input is a string, it will be interpreted as the path to a file or folder.\nAsset's and Archives change depending on their contents. This means that a change\nto the underlying file/folder will result in a \"replace\" operation for the Copy.",
```

Grammar fix.

```suggestion
                    "description": "The file/folder to be copied.\nIf the input is an Asset, it will be interpreted as the contents of a file.\nIf the input is an Archive, it will be interpreted as the contents of a folder.\nIf the input is a string, it will be interpreted as the path to a file or folder.\nAssets and Archives change depending on their contents. This means that a change\nto the underlying file/folder will result in a \"replace\" operation for the Copy.",
```

---

Question: The capitalization here is slightly different than I would expect, and I want to ensure we're clear for someone who might not know the system as well to understand the docs. Assets and Archives are, in this case, Pulumi concepts, correct? It's noticeable because we are not overwriting the general sense of the term `string` in comparison. And Copy, as well? I would refer to the action in full in the latter case (`CopyFile`) to be clear:

`Object` isn't exactly the right term as I know we use languages that are not object-oriented, but hopefully this suggestion makes clear what I'm trying to do:

```suggestion
                    "description": "The file/folder to be copied.\nIf the input is an Asset object, it will be interpreted as the contents of a file.\nIf the input is an Archive object, it will be interpreted as the contents of a folder.\nIf the input is a string, it will be interpreted as the path to a file or folder.\nAsset objects and Archive objects change depending on their contents. This means that a change\nto the underlying file/folder will result in a \"replace\" operation for the CopyFile call.",
```

---

I've heard them referred to as "types", and I think that language is universal. Thanks for pointing this out!